pkgs/tools/virtualization/awsebcli/default.nix

```diff
@@ -52,7 +52,7 @@ with localPython.pkgs; buildPythonApplication rec {
       --replace "botocore>1.23.41,<1.24.0" "botocore>1.23.41,<2" \
       --replace "pathspec==0.9.0" "pathspec>=0.10.0,<0.11.0" \
       --replace "colorama>=0.2.5,<0.4.4" "colorama>=0.2.5,<=0.4.6" \
-      --replace "termcolor == 1.1.0" "termcolor>=2.0.0,<2.1.0"
+      --replace "termcolor == 1.1.0" "termcolor>=2.0.0,<2.2.0"
```

Comment:
Next time just pin it to 3 to make maintenance easier

Reply:
will do